ui/src/models/workflows.ts

```diff
@@ -916,15 +916,17 @@ export interface WorkflowStep {
     templateRef?: TemplateRef;
 }
 
-export type NodePhase = 'Pending' | 'Running' | 'Succeeded' | 'Skipped' | 'Failed' | 'Error';
+export type NodePhase = 'Pending' | 'Running' | 'Succeeded' | 'Suspended' | 'Skipped' | 'Failed' | 'Error';
 
 export const NODE_PHASE = {
     PENDING: 'Pending',
     RUNNING: 'Running',
     SUCCEEDED: 'Succeeded',
     SKIPPED: 'Skipped',
     FAILED: 'Failed',
-    ERROR: 'Error'
+    ERROR: 'Error',
+    // this is a fake phase - no actual node even has this
+    SUSPENDED: 'Suspended'
```

Comment:
Minor:

Could we add a comment here as to why this is needed? Something like "this is used to be able to filter suspended nodes in the DAG view". Might make is easier to tell why this exists only in the UI and not in the back-end code